### DISCUSSION TITLE
code blocks on htmx.org are unreadable

### DISCUSSION THREAD

**Discussion by m9xiuz** [bigskysoftware/htmx#739] [General]
Hi. I was reading https://htmx.org/essays/a-response-to-rich-harris/ and opened hx-push-url link https://htmx.org/attributes/hx-push-url/ but I couldn't read anything because the code is unreadable with js disabled:
![image](https://user-images.githubusercontent.com/61184451/147459637-21dc3b98-9157-4b2f-98a1-20bae7383daf.png)
And home button also doesn't work.
If you can't make even your own website useful and functional, I think your web technology doesn't worth attention.

**Comment by ocharles** [comment]
>  If you can't make even your own website useful and functional, I think your web technology doesn't worth attention.

This is unnecessary and uncalled for.
